packages/config/src/doc/IProfArgAttrs.ts

```diff
@@ -35,4 +35,7 @@ export interface IProfArgAttrs {
 
     /** Whether the argument value is stored securely */
     secure?: boolean;
+
+    /** Whether the argument value is defined in the schema file */
+    offSchema?: boolean;
```

Comment:
The name `offSchema` sounds weird to me - typically "off" is used in a boolean context for on/off. Would `inSchema` make sense?

Reply:
the JSDoc is backwards 😋 
The idea is to identify which properties are not defined in the schema (true), otherwise, the property `offSchema` would be `Falsy` (undefined, false, ...)

Reply:
@t1m0thyj I agree that the name is a bit counter intuitive. @zFernand0 though the use case for this is for ZE to tell what isn't on the schema, I think it would still be alright to change the property name then search for properties not listed in the schema with `'inSchema': false` 

If that's alright with you, I'll make the change

Reply:
sorry for the delay, and yeah, I think that makes sense 😋